baseplate/frameworks/queue_consumer/deprecated.py

```diff
@@ -148,18 +148,22 @@ def get_batch(self, max_items: int, timeout: Optional[float]) -> Sequence[Messag
             block = False
         else:
             block = True
-
         batch = []
         retry_policy = RetryPolicy.new(attempts=max_items, budget=timeout)
         for time_remaining in retry_policy:
-            item = self.work_queue.get(block=block, timeout=time_remaining)
+            item = self._get_next_item(block=block, timeout=time_remaining)
             if item is None:
                 break
```

Comment:
I'm not really sure why we introduce a new function here when it's pretty clear inlined:

```suggestion
            try:
                item = self.work_queue.get(block=block, timeout=time_remaining)
            except queue.Empty:
                break
```
```

Reply:
So, that’s what I did originally but the thing about this code it it’s _not_ actually the same behavior as it was originally. 

One of the tests started failing because it was relying on the old behavior, which is checking `item is None`.  It’s probably debatable whether that’s even the behavior we actually want but for something like this, I erred on the side of just preserving the old behavior.

Reply:
yuck. ok, thanks for the explanation.